### ISSUE TITLE
Accessibility: "Video preview thumbnail"

### ISSUE THREAD

**Issue by bmivzkrp** [TeamNewPipe/NewPipe#7583]
### Checklist

- [x] I am using the latest version - x.xx.x 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Actual behavior

Before the title of each subscription and each video, I hear the phrase:

"Video preview thumbnail"

### Expected behavior

The screen reader does not announce this phrase. That is, the screen reader immediately announces the name of the subscription, or the name of the video.

### Device info

 - Android version/Custom ROM version:
 
6.0

 - Device model:

Impression ImSmart-C551

**Comment by bmivzkrp** [comment]
During playback in the background, in the lower left corner of the screen, a button is displayed, which, if I understood correctly, provides access to information about the content being played. When I focus on this button, I also hear this phrase. That is, the button is named:

"Video preview thumbnail"

Please name it differently. For example - "Info" or something like that. The title should be as short as possible, but at the same time it should be clear, that is, to reflect the essence.

I can t tell if this button is unsigned or its real name, so sorry if I created a duplicate again.

**Comment by bolshoytoster** [comment]
I can change this, but I'll likely also have to change it for each of the 68 languages supported.

**Comment by bmivzkrp** [comment]
@bolshoytoster I can say for sure that the NewPipe translation into Ukrainian is not the worst. That is, translators will translate everything correctly. If not, I will try to contact them. The same goes for other languages: if a blind user thinks that the names of the elements have been translated incorrectly into their language, they can turn to translators. Or fix it yourself, because WebLate is at least partially accessible.

**Comment by bmivzkrp** [comment]
@bolshoytoster I meant the names of the elements in English. Of course, for other languages, everything will depend on the translators and the activity of blind users.

**Comment by bolshoytoster** [comment]
@bmivzkrp alright, I'll just change the english version.

**Comment by bolshoytoster** [comment]
I've opened a [pr](https://github.com/TeamNewPipe/NewPipe/pull/7584).

**Comment by bmivzkrp** [comment]
@bolshoytoster Many thanks for that.

**Comment by bmivzkrp** [comment]
@bolshoytoster For Ukrainian:

Інформація

**Comment by bolshoytoster** [comment]
@bmivzkrp thank you.

**Comment by litetex** [comment]
In my opinion this works as expected.
A thumbnail of the video is displayed there - so that's also the reason why it's read.

**Comment by bmivzkrp** [comment]
@litetex This phrase slows down a blind user with a subscription list or content list. I assure you that it is very tiring when you constantly hear this phrase. In addition, it is not useful for the blind user.

**Comment by bolshoytoster** [comment]
We could add a setting to enable/disable long descriptions.

**Comment by bmivzkrp** [comment]
@bolshoytoster I forgot to add. In the NewPipe settings, I ve turned off thumbnail downloads, and I still hear that phrase. Maybe it s just worth improving, if possible. If the user disables thumbnail downloads, he does not hear this phrase.

**Comment by bmivzkrp** [comment]
It seems that this phrase is not only in the list of subscriptions and in the list of content, but also in other places. For example - when we activate a subscription, ie open it, or when we open information about a video.
Now I m sure it s necessary to get rid of it everywhere if the user turns off thumbnail downloads. I hope it doesn t hurt people who see.

**Comment by litetex** [comment]
> This phrase slows down a blind user with a subscription list or content list.
> ...
> In addition, it is not useful for the blind user

Disclaimer: I always (personally) question in the first place why (completely) blind users NewPipe - an app primarily built for video streaming - in the first place, but that doesn't belong here.
As far as I know accessibility should allow someone to further explain/explore parts of the app. E.g. if you click on something (e.g. text you can't read because it's too small) and then text-to-speech says you what is there. That's how I understand accessibility.

> We could add a setting to enable/disable long descriptions.

I don't know how this is related to this issue. Marking as off-topic.

> ...I ve turned off thumbnail downloads, and I still hear that phrase...

I think you refer to the "Load thumbnails" setting.
Works as expected: The thumbnails are still shown but no images are loaded from the servers

> It seems that this phrase is not only in the list of subscriptions and in the list of content, but also in other places.

It's used everywhere here: https://github.com/TeamNewPipe/NewPipe/search?q=%22%40string%2Flist_thumbnail_view_description%22

----

Anyway I think the most simple solution would be removing the accessibility string for the thumbnails altogether. Other apps I have installed on my phone (e.g. the settings app) do this.

**Comment by bmivzkrp** [comment]
@litetex 1. Blind users listen to audio, ie enjoy the opportunity to play in the background. Sometimes blind users can show something to sighted users, and in this case they play the video.
Also, blind users do not refuse to read subtitles. But I haven t researched this feature, so I can t say it s there with accessibility.
2. There are different cases. Now the information about the sketches is not useful for me. But maybe in some case, for a short time, I will need this information. So I think there has to be some setting that controls the behavior of the thumbnail display.

**Comment by bmivzkrp** [comment]
Browsers for both Linux and Android have the ability to turn off the download and display of thumbnails and images. This was definitely done for people who see, because in fact there are very few developers who try to make their applications user-friendly and for people who see and for blind users. So if possible, and if the developers do not mind taking into account the needs of the blind, it is better to make the display of thumbnails customizable. Thumbnails will be displayed by default, but a blind user can turn it off if they don t need it, and may not receive information they don t need.
We should also keep in mind that there are people who have poor eyesight. They may change this setting more often than I do. They use both their eyes and the screen reader.

**Comment by bmivzkrp** [comment]
I asked a few people who see. They replied that they had not thought about it, but in general they agreed that it should be possible not to show thumbnails in the same way as we can turn off comments.

**Comment by litetex** [comment]
@bmivzkrp 
I removed the accessibility description for the thumbnails in #7619. You may check it out 😄

**Comment by bmivzkrp** [comment]
@litetex
Sorry, I can t build an APK right now, but I understand what you did.

**Comment by bmivzkrp** [comment]
@litetex Thank you very much. It seems to work as expected.
By the way: in this PR. The button that appears in the lower left corner 
of the screen during content playback is now named from my screen reader 
s point of view:
"Overlay thumbnail".
